src/lib/iptables.ts

```diff
@@ -291,3 +303,31 @@ async function applyRules(rules: Rule | Rule[], adaptor: RuleAdaptor) {
 
 	await adaptor(processedRules);
 }
+
+class StdError extends TypedError {}
+
+export async function getRulePosition(
+	chain: string,
+	ruleMatch: string,
+): Promise<number> {
+	const cmd = `iptables -L ${chain} --line-numbers`;
```

Comment:
Let's add `-n` flag here after the `-L` argument, because without it sometimes I see significant slowdowns in iptables output due to reverse DNS lookup.

Reply:
Mentioned [here](https://ipset.netfilter.org/iptables.man.html) as well on the `-L` docs.

Reply:
Added `-n` in [this line](https://github.com/balena-os/balena-supervisor/pull/2135/commits/01faa2cba7c902aad28e5e5f0ed2d1fb17645620#diff-fa58e353d5a5691026b53449e0a68869fbfda197dc1c2b62b1cc2e1f56303272R313).